docs/site/migration/overview.md

```diff
@@ -55,8 +55,8 @@ require migration:
    Learn how to convert your boot scripts to lifecycle observers in
    [Migrating boot scripts](boot-scripts.md).
 
-2. To make your application secure, LoopBack 3 offers several ways how to
-   implement authentication and authorization. Check out
+2. To make your application secure, LoopBack 4 offers several ways to implement
```

Comment:
I think that it was intended to be LoopBack 3, because if you go to the [Authentication & authorization page](https://loopback.io/doc/en/lb4/migration-auth-overview.html), it lists the ways you can migrate from LB3 auth to LB4. 

Maybe we could fix the wording to make it more clear, though? E.g. "for an overview and links to further resources" -> "for an overview and links to further resources on how to migrate to LoopBack 4 authentication/authorization"

What do you think?

Reply:
Yep, I completely misunderstood the intent. And now that I re-read the whole thing, I think the line:

> To make your application secure, ...

which is sort of there to look LB3 look better, threw me off that it should have been LB4. 😄 

Anyways. I like your suggestion, I will update the commit.

Thanks for the quick review.